test/lib/builder/builder.js

```diff
@@ -462,37 +567,66 @@ test.serial("Cleanup", async (t) => {
 	t.deepEqual(executeCleanupTasksStub.callCount, 2, "Cleanup called twice");
 });
 
-const applicationATree = {
```

Comment:
What happened here? Was applicationATree switched with libraryDTree?

Reply:
- went with {Array.<string|RegExp>} for the JSDoc
- removed fallback from project.dependencies
- I wanted to use libraryDTree in applicationATree, so I moved libraryDTree above applicationATree and changed the dependency in applicaitonATree to include libraryDTree

Reply:
👍 nice